lib/node_modules/@stdlib/math/base/special/sincos/lib/main.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+// MODULES //
+
+var computeSincos = require( './sincos.js' );
+
+
+// MAIN //
+
+/**
+* Simultaneously computes the sine and cosine of a number.
+*
+* @param {array} [out] - destination array
```

Comment:
Object types should always be capitalized, e.g. `@param {Array}`,  `@param {Function}`, and `@param {Object}`.